parse.py

```diff
@@ -49,11 +50,12 @@
 						textBlock = '{} {}\n\n'.format('#' * HSIZE, header.group(1))
```

Comment:
This needs to target group 2 since everything is shifted by one to match the `=*` as group 1.